### PR TITLE
add_social_media

### PR DESCRIPTION
**Pull request by hubert-hernoga**

### CODE REVIEW

File: shop/templates/base.html [hunk 1]
```diff
@@ -576,7 +576,7 @@ <h3 class="widget-title">Get In Touch With Us</h3>
...
+                            <p><a class="navi-link-light" href="mailto:pawelmarlykizomba@gmail.com">pawelmarlykizomba@gmail.com</a></p><a class="social-button shape-circle sb-facebook sb-light-skin" href="https://www.facebook.com/pawelmarly/"><i class="socicon-facebook"></i></a><!-- <a class="social-button shape-circle sb-twitter sb-light-skin" href="#"><i class="socicon-twitter"></i></a>--> <a class="social-button shape-circle sb-instagram sb-light-skin" href="https://www.instagram.com/pawel_marly_kizomba/"><i class="socicon-instagram"></i></a><!-- <a class="social-button shape-circle sb-google-plus sb-light-skin" href="#"><i class="socicon-googleplus"></i></a>-->
```
Comment (pawel-krysiak) [inline shop/templates/base.html]: this is opened in the same or other page ?

Comment (hubert-hernoga) [reply shop/templates/base.html]: small new window on this same page

Comment (pawel-krysiak) [reply shop/templates/base.html]: We open small facebook on our page? How does that look like ?

Comment (hubert-hernoga) [reply shop/templates/base.html]: I meant an email. Facebook and Instagram opens in new browser windows.

Comment (pawel-krysiak) [reply shop/templates/base.html]: @hubert-hernoga are you sure that in those links as well? can you check it?

Comment (hubert-hernoga) [reply shop/templates/base.html]: Done. They work properly  :)

**Pull request merged**